inspectit-oce-documentation/build.gradle

```diff
@@ -17,7 +17,8 @@ asciidoctor {
     attributes 'inspectit-oce-version': project.version,
             'opencensus-version': "${openCensusVersion}",
             'gradle-rootdir-absolute': rootDir.absolutePath,
-            revnumber: project.version
+            revnumber: project.version,
+            toc: 'left'
```

Comment:
naaahh I did no like the left TOC due to the length of the headings, but fine..

Reply:
I agree with Alex' version here, makes it much easier to navigate